packages/gatsby-telemetry/src/telemetry.ts

```diff
@@ -475,7 +475,10 @@ export class AnalyticsTracker {
     this.metadataCache[event] = Object.assign(cached, obj)
   }
 
-  addSiteMeasurement(event: string, obj): void {
+  addSiteMeasurement(
+    event: string,
+    obj: ITelemetryTagsPayload["siteMeasurements"]
+  ): void {
```

Comment:
I can revert this part - just though some extra type safety won't hurt

Reply:
This is great!